### ISSUE TITLE
Error running readme.js in /samples

### ISSUE THREAD

**Issue by anandgeorge** [wdavidw/node-sigar#7]
Get the following error trying to run the readme.js example

Error: Cannot find module './build/Release/sigar'

Sigar does not seem to be in this directory.

Please help.

**Comment by wdavidw** [comment]
george,

how did u install the module and which version of node are u using?

d.

**Comment by wdavidw** [comment]
i forgot to also ask you about your os

**Comment by anandgeorge** [comment]
Hi David,

My OS is Ubuntu 10.04. Used sudo npm install sigar -g. Node version 0.6.8.

Thanks,

On Fri, Feb 17, 2012 at 1:47 AM, Worms David <
reply@reply.github.com

**Comment by wdavidw** [comment]
it's close from my environment, i'll try to look at this tomorrow. have you try just `npm install sigar`, without the global argument?

**Comment by anandgeorge** [comment]
Have tried that option too. Get the same error in both cases. Noticed that
I was getting a similar error installing another module too. That seems to
have been fixed using Node v0.4.7 at runtime. But this option also throws
up the same error with sigar.

On Sat, Feb 18, 2012 at 4:36 AM, Worms David <
reply@reply.github.com

**Comment by wdavidw** [comment]
Yes, at that time, it was because the rename to final destination to `Release`. The binding was generated but just in a different directory. Do you have sth in build, how does it looks like?

**Comment by Nessworthy** [comment]
Is this repo dead?
Following `npm install sigar --save` yields the missing build directory error. The repo contains no actual build directory or compiled code, and I don't fancy having to mess around with it when there's no installation instructions beyond that one line.

**Comment by wdavidw** [comment]
no using it at the moment but i wouldn't consider it dead. build system need to be aligned with node-gyp

**Comment by andreamishon** [comment]
Hello, I am also facing this issue: Error: Cannot find module './build/Release/sigar'. Has there been a fix or is there a workaround?

**Comment by wdavidw** [comment]
this was build against node 0.6 or something similar, need some work for current nodejs
